src/field.ts

```diff
@@ -149,7 +142,7 @@ export function isMessage(fieldDescriptor: descriptor.FieldDescriptorProto) {
 }
 
 /**
- * @param {descriptor.FieldDescriptorProto} fieldDescriptor
+ * @param {descriptor.FieldDescriptorProto} fieldDescriptorń
```

Comment:
Take back?

Reply:
I have no idea how the strange character appeared in the file.  I will take back that change.